src/Entity/Server.php

```diff
@@ -498,10 +504,90 @@ protected function getValidationRules() {
         return [];
       }
 
-      return array_values(DocumentValidator::defaultRules());
+      $rules = array_values(DocumentValidator::defaultRules());
+      if ($this->getDisableIntrospection()) {
+        $rules[DisableIntrospection::class] = new DisableIntrospection();
+      }
+      if ($this->getQueryDepth()) {
+        $rules[QueryDepth::class] = new QueryDepth($this->query_depth);
```

Comment:
ah, we forgot to add the new properties to the class and document them.

Reply:
@klausi , my bad, it isn't a property, I forgot to use the appropriate getter. Will fix this.